.openhands/skills/visual-explainer/references/libraries.md

```
@@ -0,0 +1,482 @@
+  mermaid.initialize({
+    startOnLoad: true,
+    theme: 'base',
+    look: 'handDrawn',
+    layout: 'elk',
+    themeVariables: { /* same as above */ }
+  });
+</script>
+```
+
+Or set it per-diagram via frontmatter:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)